apps/condo/domains/common/constants/currencies.js

```diff
@@ -180,6 +180,22 @@ const ISO_CODES = [
     'ZWL',
 ]
 
+/**
+ * TODO(DOMA-3344): replace with intl ?
+ */
+const CURRENCY_SYMBOLS = {
+    CNY: '¥',
+    EUR: '€',
+    INR: 'र',
+    JPY: '¥',
+    RUB: '₽',
+    USD: '$',
+}
+
+const DEFAULT_CURRENCY_CODE = 'RUB'
```

Comment:
I think we may already have this const in billing or acquiring domain 

Just a note